python/mkdocs/docs/feature-support.md

```diff
@@ -21,21 +21,21 @@ The goal is that the python library will provide a functional, performant subset
 
 ## Metadata
 
-| Operation                | Java  | Python |
-|:-------------------------|:-----:|:------:|
-| Get Schema               |    X  |   X    |
-| Get Snapshots            |    X  |   X    |
-| Plan Scan                |    X  |   X    |
-| Plan Scan for Snapshot   |    X  |   X    |
-| Update Current Snapshot  |    X  |        |
-| Create Table             |    X  |   X    |
-| Rename Table             |    X  |   X    |
-| Drop Table               |    X  |   X    |
-| Alter Table              |    X  |        |
-| Set Table Properties     |    X  |        |
-| Create Namespace         |    X  |   X    |
-| Drop Namespace           |    X  |   X    |
-| Set Namespace Properties |    X  |   X    |
+| Operation                | Java | Python |
```

Comment:
I checked and this looks good: 
![image](https://user-images.githubusercontent.com/1134248/229110518-83418ffa-c359-49ca-8a13-7b661d6e46c8.png)